### PR TITLE
Release minor 0.18

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

**Review by schencej — APPROVED** [review]
LGTM!

File: docs/release_notes/v0.18.0.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+v0.18.0
+=======
+This minor release primarilly adds two new optional plugins for
```
Comment (joshanderson-kw) [inline docs/release_notes/v0.18.0.rst:3]:
```suggestion
This minor release primarily adds two new optional plugins for
```

**Review by joshanderson-kw — APPROVED** [review]
LGTM!

**Comment by Purg** [comment]
Oh hey, the publish action shows up in the checks list :)

**Pull request merged**